helpers/layout.py

```diff
@@ -83,12 +83,12 @@ def _dcc_radio(name, suffix):
             style={"display":"inline-block"}),
         "documentation."],
             style={"display":"inline-block"}),
-        html.Div(["If you are running in the Leap IDE, see the ",
-        Link(children=[html.Div("Leap IDE dumentation")],
-            href="https://docs.dwavesys.com/docs/latest/doc_ide_user.html",
-            style={"display":"inline-block"}),
-        "documentation"],
-            style={"display":"inline-block"}),])]
+        html.Div(['If you are running in an online IDE, see the ',
+        Link(children=[html.Div('system documentation')],
+            href='https://docs.dwavesys.com/docs/latest/doc_leap_dev_env.html',
+            style={'display':'inline-block'}),
+        ' on supported IDEs.'],
+            style={'display':'inline-block'}),])]
```

Comment:
Why the change to single quotes? I'd say keep double for consistency.

Reply:
Updated, thanks